css-three-column-layout/styles.css

```diff
@@ -0,0 +1,63 @@
+body {
+  background-color: #1996da;
+  margin: auto;
+  width: 70%;
```

Comment:
Instead of setting `margin` and `width` on the `body`, you should apply these to the `container` class